Storage/src/Connection/Rest.php

```diff
@@ -689,18 +676,6 @@ protected function crc32cExtensionLoaded()
         return extension_loaded('crc32c');
     }
 
-    /**
-     * Check if hash() supports crc32c.
-     *
-     * Protected access for unit testing.
-     *
-     * @return bool
-     */
-    protected function supportsBuiltinCrc32c()
```

Comment:
Because this is protected, we can't remove it without the BC check complaining (even though it's probably harmless to do so).

Just mark it as deprecated and have it return false:

```php
    /**
     * Check if hash() supports crc32c.
     *
     * @deprecated
     * @return bool
     */
    protected function supportsBuiltinCrc32c()
    {
        return false;
    }
```

This is why we shouldn't make methods protected for testing!!